challenges/Ayush/medium-task-4-build-markdown-to-html-previewer/index.html

```
@@ -0,0 +1,173 @@
+                <div class="guide-item">
+                    <h3>Code Blocks</h3>
+                    <pre><code>\`\`\`javascript
+function hello() {
+    console.log("Hello, World!");
+}
+\`\`\`</code></pre>
+                </div>
+
+                <div class="guide-item">
+                    <h3>Blockquotes</h3>
+                    <pre><code>> This is a blockquote
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)